chainer/datasets/kuzushiji_mnist.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import numpy
+
+import chainer
+from chainer.dataset import download
+from chainer.datasets._mnist_helper import make_npz
+from chainer.datasets._mnist_helper import preprocess_mnist
+
+
+_kuzushiji_mnist_labels = [('o', u'\u304A'), ('ki', u'\u304D'),
+                           ('su', u'\u3059'), ('tsu', u'\u3064'),
+                           ('na', u'\u306A'), ('ha', u'\u306F'),
+                           ('ma', u'\u307E'), ('ya', u'\u3084'),
+                           ('re', u'\u308C'), ('wo', u'\u3092')]
```

Comment:
Could you comment on why the Unicode hiragana should be included (and not just the romaji)?

Reply:
The unicode codepoints aren't essential, but for those unfamiliar with the Japanese language an easy way to print and refer to the characters might be appreciated (i.e. seeing something classified as 'ma' might not mean much, but being able to see the original character easily could help). What do you think? Is consistent with Fashion MNIST that important in this context?

Reply:
I see, thanks for the explanation. So you mean for actual visual verification? Consistency matters simply in terms of how much the users has to rewrite when switching from Fashion MNIST to Kuzushiji MNIST. Maybe this is not that much of an issue, regarding the fact that labels are mostly meant for practical visualizations in the first place.

Reply:
Let's proceed with this PR as is.